provider/go.mod

```diff
@@ -9,7 +9,7 @@ require (
 )
 
 replace (
-	github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 v2.0.0-20220824175045-450992f2f5b9
+	github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 v2.0.0-20230327102345-3fa930f86570
```

Comment:
Did you have to manually update this?

Reply:
Nope: https://github.com/pulumi/upgrade-provider/blob/2b4e08ac8d01517bb4ebbfbde8870bcf13beb104/upgrade/steps.go#L337